scripts/Game/index.js

```diff
@@ -48,7 +48,8 @@ let projectiles = [];
 let coins = [];
 let spawners = [];
 
-let buttons = [];
+let statsButtons = [];
+let deathButtons = [];
```

Comment:
We don't need to have different arrays, all of the buttons can be stored in the same array.
What we'll have to do for pausing, is make a variable in the Button class that's for scene, and only display that button if the scene = the buttons scene.